### ISSUE TITLE
Update disabled dropdown states in checkout flow

### ISSUE THREAD

**Issue by nfrgosselin** [gitcoinco/web#10008]
**User Story:**
As a grant donor
I want to see clear exception states on the checkout button
So that I know when I need to make changes to my cart

**Acceptance Criteria**
GIVEN that I am on gitcoin.co/grants/cart
WHEN all of the checkout options are unavailable with my current cart settings
THEN I see the disabled dropdown state on the “I’m ready to checkout” CTA and I can’t open the dropdown menu

GIVEN that I am on gitcoin.co/grants/cart
WHEN a subset of the checkout options are unavailable
THEN I see the Partially Disabled dropdown state on the Checkout button

**Product & Design Links:**
Figma: "[Ticketed: Disabled Dropdown](https://www.figma.com/file/8QjqOzDBd4HiE37YqcocK7/GR13?node-id=110%3A5601)" page -> Checkout flow sections 1 + 2

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
This is **not** for updating the top error banners — that will be separate story.

**Comment by nfrgosselin** [comment]
Finish #10007 before picking up this ticket

**Comment by EmHsiehGitcoin** [comment]
@thelostone-mc  Maybe we can create a multi-sig test grant and it'd be blocked by polygon!?

**Comment by nfrgosselin** [comment]
+1 — @thelostone-mc is there a better way to verify this?

I created a multisig grant (using the Gitcoin multisig) and Polygon is still showing as a valid option:
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/15041635/156086412-66738221-3475-4e22-b26e-e1ca7ed4234b.png">

**Comment by thelostone-mc** [comment]
Hmm it looks like the fix doesn't work. This would have to be debugged and fixed

**Comment by EmHsiehGitcoin** [comment]
just saw #10319 recording looks good to me!
